src/components/EnvironmentBlock.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    <h1>Try to use calculator</h1>
+    <button class="button_environment">View</button>
+  </div>
+</template>
+
+<script>
+export default {};
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
+
+div {
+  --main_block_paddings_big_screen: 80px;
```

Comment:
Move all project related variables to one place, for example file 'variables.scss'

Reply:
moved